week-5/Homework/mandatory/3-project/index.html

```diff
@@ -65,7 +65,7 @@ <h1 class="display-3">Bikes for Refugees</h1>
 					<button id="greenBtn" type="button" class="btn btn-primary colorButton">Green</button>
 					<br>
 					<h3>Register with us today:</h3>
-					<form >
+					<form onsubmit="return formValidate()">
```

Comment:
Instead of changing the HTML here is there another way you could have done this?

Hint:
https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Event_handlers

Reply:
thank you